src/actions.ts

```diff
@@ -1090,7 +1090,7 @@ export function GetActions(instance: InstanceBaseExt<ZoomConfig>): CompanionActi
 				}
 				sendActionCommand(sendToCommand)
 				// Also update locally
-				let ZoomID = action.options.user as number
+				const ZoomID = action.options.user as number
```

Comment:
Just saw that in other parts to hahaha

Reply:
@JeffreyDavidsz That was a linting fix.  With the linting changes, I would run yarn prepare and install the husky hooks so that it auto runs the linter before it will do a git commit.

Reply:
Yeah what I meant was that the code came from me. Using let more that const ;-)